### PR TITLE
Create a database based on a diagram

### PR DESCRIPTION
**Pull request by aliabbani**
## Create a database based on a diagram

In this milestone we:

- Created a file named schema_based_on_diagram.sql where we implemented the database from the diagram.
- Created a join table for many to many relationships named history_treatment_connection
- added the FK indexes.

### CODE REVIEW

File: schema_based_on_diagram.sql [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+  treatment_id INT,
+  history_id INT,
+  PRIMARY KEY (id ),
+  CONSTRAINT FK_history_treatment_connection.treatment_id
+    FOREIGN KEY (treatment_id)
+      REFERENCES treatments(id),
+  CONSTRAINT FK_history_treatment_connection.history_id
+    FOREIGN KEY (history_id)
+      REFERENCES medical_histories(id)
+);
```
Comment (DeliceLydia) [inline schema_based_on_diagram.sql]: - [ ] Please create indexes on all FOREIGN KEYS for example:
`CREATE INDEX ON medical_histories (patient_id);`

**Review by DeliceLydia — CHANGES_REQUESTED** [review]
## Hi Team, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Required Changes ♻️
_Check the comments under the review._



### Cheers and Happy coding!👏👏👏

Feel free to leave a question, or a comment on the PR thread if something is not clear 100%. 

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: schema_based_on_diagram.sql (hunk 1 above)
Comment (HENRYKC24) [reply schema_based_on_diagram.sql]: Thank you so much, @DeliceLydia. We will affect those changes as you pointed out.

**Review by Emmyn5600 — APPROVED** [review]
Hi @aliabbani  and @HENRYKC24 

Good job so far! 👏🏼

## STATUS: APPROVED :muscle::clap:

Congratulations! :tada:
Your project is complete!  It's time to merge it :shipit:
Great work on implementing the changes requested by the previous reviewers.

 #### Optional suggestions
 
 - N/A
 
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

#### _**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_
------------------------------------------------------------------------------------------------------------------------------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Comment by HENRYKC24** [comment]
Thank you, @Emmyn5600 and @DeliceLydia. We greatly appreciate your efforts.

**Pull request merged**